src/devices/device_models/DC_branches.jl

```diff
@@ -51,11 +51,13 @@ end
 
 function branch_rate_constraint(ps_m::CanonicalModel,
                                 devices::PSY.FlattenIteratorWrapper{B},
-                                device_formulation::Type{HVDCLossless},
-                                system_formulation::Type{PM.DCPlosslessForm}) where {B <: PSY.DCBranch}
+                                device_formulation::Type{D},
+                                system_formulation::Type{S}) where {B <: PSY.DCBranch,
+                                                                    D <: AbstractDCLineForm,
+                                                                    S <: PM.DCPlosslessForm}
```

Comment:
Why changed from `system_formulation::Type{PM.DCPlosslessForm}` back to the abstract `S <: PM.DCPlosslessForm`? It should be `AbstractActivePower`?

Reply:
this now catches the `PM.NFAForm` in [this test](https://github.com/NREL/PowerSimulations.jl/pull/188/files#diff-148cdaa0ecb52b016c6b7ed0ce62d7f4L143)